web/templates/setting/form.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= form_for @conn, @action, [as: :settings, method: :put], fn f -> %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for svc <- @system_variable_changesets do %>
+        <tr>
+          <td> <%= svc.data.key %> </td>
+
+          <%= inputs_for f, integer_to_atom(svc.data.id), fn ff -> %>
```

Comment:
Ok I didn't know that. I read this article http://blog.pemedeiros.com/elixir-atoms-are-everywhere/ which says "The Erlang vm stores the Atom inside a table internally and has a default limit of 1048576 atoms per instance. This value can be configured, but in the end it is not an optimization that we want to make."
Honestly it's a very stupid limitation, but in any case I am still thinking about other forms that are using atoms. So in any form that one uses inputs_for, data gets sub-grouped by atoms which is referred to rows.

I start checking my solution to see how else I can change it